doc/src_usermanual/techrep_main.tex

```diff
@@ -669,7 +669,7 @@ \subsubsection{The C++ interface}
 			      const double* lambda, bool new_lambda,
 			      const long long& nsparse, const long long& ndense, 
 			      const int& nnzHSS, int* iHSS, int* jHSS, double* MHSS, 
-			      double** HDD,
+			      double* HDD,
 			      int& nnzHSD, int* iHSD, int* jHSD, double* MHSD);
 \end{lstlisting} 
 

```

Comment:
I think it's better if we add another note to mention that only a triangular part is needed to pass into hiop